x/nft/keeper/keeper.go

```diff
@@ -159,3 +159,22 @@ func (k Keeper) BurnNFT(ctx sdk.Context, denomID, tokenID string, owner sdk.AccA
 
 	return nil
 }
+
+// BurnNFTUnverified deletes a specified NFT without verifying if the owner is the creator of denom
+// Needed for IBC transfer of NFT
+func (k Keeper) BurnNFTUnverified(ctx sdk.Context, denomID, tokenID string, owner sdk.AccAddress) error {
+	if !k.HasDenomID(ctx, denomID) {
+		return sdkerrors.Wrapf(types.ErrInvalidDenom, "denom ID %s not exists", denomID)
+	}
+
+	nft, err := k.IsOwner(ctx, denomID, tokenID, owner)
```

Comment:
not sure if we need check owner again after check in https://github.com/crypto-org-chain/chain-main/blob/f598631b4b71636e4fb354a8a49683570b4d165b/x/nft-transfer/keeper/packet.go#L98

Reply:
This function can be called from other places also.